public/js/portfolio.js

```diff
@@ -11,6 +11,9 @@ let portfolio = createPage(
             }
         ]
     },
+    [],
+    {},
+    null,
```

Comment:
Can these new arg moved to last with default value so no need to fill when not needed?

Reply:
Sure, will fix it in a moment

Reply:
Done, you can check now.